sdk/src/main/java/co/omise/android/api/Config.kt

```diff
@@ -0,0 +1,58 @@
+package co.omise.android.api
+
+import android.os.Build
+import co.omise.android.BuildConfig
+
+/**
+ * Config class bundles configuration values supplied to the [Client] constructor.
+ *
+ * @param apiVersion The API version to use.
+ * @param publicKey  The key with the `pkey_` prefix.
+ *
+ * @see Client
+ */
+class Config(
+        private val apiVersion: String?,
```

Comment:
I think we should pin to the latest API version here. Since we need the schema to be persistent to parse the result.

Reply:
Same ☝️

Reply:
The latest API version is pinned is being pinned through `EndPoint`, is that not ok?

Reply:
Since we using fixed API version I don't think it necessary to have `apiVersion`. 🤔

Reply:
This just gives up flexibility if needed in the future, and it's pinning the version anyway so what we want is achieved. So I don't think it's that necessary to make changes here.

Reply:
I understand your point. How's about demote the Config to internal access level and set the `apiVersion` to be the predefined hard code value? Also make it NonNull as well

Reply:
Hmmm that's a good idea, something like this `private val apiVersion: String = "xxxxxx"`?

Reply:
Yup :)

Reply:
Ok sorted 👌 🙏

Reply:
👌